tests/Content/TestCase.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace Kirby\Content;
+
+use Kirby\Cms\App;
+use Kirby\Filesystem\Dir;
+use Kirby\TestCase as BaseTestCase;
+
+class TestCase extends BaseTestCase
+{
+	public const TMP = KIRBY_TMP_DIR;
+
+	protected $app;
```

Comment:
`Todo`: Could you please rebase the PR against `v5/develop`? Merging #6474 broke test classes like this one because of the incompatible app property definition. Fixed that on `v5/develop` for the previous PRs we merged, but this PR would reintroduce the issue.